apps/qubit/modules/settings/templates/pageElementsSuccess.php

```diff
@@ -54,7 +54,7 @@
               <?php echo $form->toggleDigitalObjectMap->label('Digital object map')->renderLabel() ?>
               <?php if (!$googleMapsApiKeySet): ?>
                 <div class="description">
-                  <?php $securitySettingsLinkHtml = link_to(__('Security'), array('module' => 'settings', 'action' => 'security')) ?>
+                  <?php $securitySettingsLinkHtml = link_to(__('global'), array('module' => 'settings', 'action' => 'global')) ?>
                   <?php echo __('This feature will not work until a Google Maps API key is specified on the %1% settings page.', array('%1%' => $securitySettingsLinkHtml)) ?>
```

Comment:
I fixed it but change isn't being reflected by GitHub in the over snippet (prolly as I rebased).